kalliope/core/ConfigurationManager/SettingLoader.py

```diff
@@ -845,3 +659,54 @@ def _get_start_options(settings):
 
         logger.debug("Start options: %s" % options)
         return options
+
+    @staticmethod
+    def _get_hooks(settings):
+        """
+        Return hooks settings
+        :param settings: The YAML settings file
+        :return: A dict containing hooks
+        :rtype: dict
+        """
+
+        try:
+            hooks = settings["hooks"]
+
+        except KeyError:
+            # if the user haven't set any hooks we define an empty dict
+            hooks = dict()
+            hooks["on_start"] = None
```

Comment:
Maybe it can be usefull to define a global variables for each hook and resuse this variables instead of the real string each time ?

Reply:
you are right. much cleaner.